src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -191,11 +194,6 @@ remove all whitespace including the final carriage return.
 For the `-key` flag use the key created earlier and stored in the media.key
 file. However, when entering it on the command line, be sure you've removed its
 whitespace. For the `-key_id` flag repeat the key value.
-
-> **Note:** Technically, the key ID is supposed to be either the first 8 OR the
-> first 16 hex digits of the key. Since packager requires the key to be 16
-> digits and does not allow a 32 digit key, both flags use the same value.
-
  
     packager \
       input=myvideo.mp4,stream=audio,output=glocka.m4a \
```

Comment:
--enable_fixed_key_decryption is not needed below.

'-key_id INSERT_KEY_HERE' should be '-key_id INSERT_KEY_ID_HERE'